src/java/org/apache/cassandra/io/sstable/format/trieindex/TrieIndexFormat.java

```diff
@@ -331,6 +333,7 @@ static class TrieIndexVersion extends Version
             hasOriginatingHostId = version.matches("(a[d-z])|(b[b-z])") || version.compareTo("ca") >= 0;
             hasMaxColumnValueLengths = version.matches("b[a-z]"); // DSE only field
             correspondingMessagingVersion = version.compareTo("ca") >= 0 ? MessagingService.VERSION_SG_10 : MessagingService.VERSION_3014;
+            hasExplicitlyFrozenTuples = version.compareTo("cc") >= 0 && version.compareTo("da") < 0; // we don't know if what DA is going to be eventually, but it is almost certain it will not include explicitly frozen tuples
```

Comment:
We incidentally have all the versions lesser than `da` that don't exist in OSS. But if we are going to start to diverge and create our own non-OSS formats, what naming conventions would we use?

Reply:
separate discussion I guess